test/ScanDependencies/modulemap_vfs_remap.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/module-cache)
+// RUN: %empty-directory(%t/inputs)
+
+// RUN: echo "{" > %/t/inputs/vfs.json
```

Comment:
Using `split-file` would be much cleaner.

Reply:
I usually create a "template" file that has "variables" that get substituted by `sed`: [example](https://github.com/llvm/llvm-project/blob/main/clang/test/ClangScanDeps/header-search-pruning-transitive.c#L40-L55). Not sure if there's better way.

Reply:
split-file will create sub directories automatically from the paths you provide, e.g.

```
// RUN: split-file %s %t

//--- a/file1

//--- b/file2

```

For creating the vfs I would recommend piping a template through `sed` to inject the path prefix.  We do this in many clang-scan-deps tests.


```
// RUN: split-file %s %t
// RUN: sed -e "s|DIR|%/t|g" %t/build/vfs.yaml.in > %t/build/vfs.yaml

//--- build/vfs.yaml.in

{
...
        "external-contents": "DIR/build/module.modulemap",
        "name": "module.modulemap",
        "type": "file"
```

Reply:
Okay, refactored to use `split-file` and `sed`, thank you.